### PR TITLE
feat(seeding): add test seeding data

### PR DESCRIPTION
**Pull request by Phil91**
## Description

Add Test data for credentials

## Why

to provide a expired credential type and not yet valid credential type for testing purposes

## Issue

Refs: #118

## Checklist

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/ssi-credential-issuer/blob/main/docs/technical-documentation/dev-process/How%20to%20contribute.md)
- [x] I have performed [IP checks](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-04#checking-libraries-using-the-eclipse-dash-license-tool) for added or updated 3rd party libraries
- [x] I have created and linked IP issues or requested their creation by a committer
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally
- [x] I have added tests that prove my changes work
- [x] I have checked that new and existing tests pass locally with my changes
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added copyright and license headers, footers (for .md files) or files (for images)

### CODE REVIEW

**Comment by Phil91** [comment]
> could you please resolve the conflict and clarify the milestone of #118?

@evegufy will directly do

**Comment by evegufy** [comment]
@ntruchsess could you please review?

File: src/database/SsiCredentialIssuer.Migrations/Seeder/Data/use_cases.json [hunk 1]
```diff
@@ -43,5 +43,10 @@
     "id": "3793a2d9-6bc5-44f0-b952-7d3f6b747dd7",
     "name": "Demand and Capacity Management",
     "shortname": "Puris"
+  },
+  {
+    "id": "1aacde78-35ec-4df3-ba1e-f988cddcbbd9",
```
Comment (ntruchsess) [inline src/database/SsiCredentialIssuer.Migrations/Seeder/Data/use_cases.json:48]: what is this dataset good for? It's not linked anywhere

**Review by ntruchsess — CHANGES_REQUESTED** [review]

**Review by ntruchsess — APPROVED** [review]

File: src/database/SsiCredentialIssuer.Migrations/Seeder/Data/use_cases.json (hunk 1 above)
Comment (Phil91) [reply src/database/SsiCredentialIssuer.Migrations/Seeder/Data/use_cases.json:48]: linkage to use case none was added.

**Review by evegufy — APPROVED** [review]

**Pull request merged**